app/templates/components/conversations/conversation-part-closed.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="coversation-part-closed__message-wrapper">
+  <div class="conversation-part-closed__timestamps-wrapper">
+    <div class="conversation-part-closed__timestamps">
+      <span data-test-closed-at>
+  {{#if isSelf}}
+    You closed this on {{moment-format conversationPart.closedAt 'MM/DD/YYYY hh:mm:ss'}}
+  {{else}}
+    {{author.username}} closed this on {{moment-format conversationPart.closedAt 'MM/DD/YYYY hh:mm:ss'}}
```

Comment:
First of all, even if you write the test correctly, it wont pass because your component has a bug here:

It's supposed to be 

```Handlebars
{{#if isSelf}}
  You closed this on {{moment-format closedAt 'MM/DD/YYYY hh:mm:ss'}}
{{else}}
  {{author.username}} closed this on {{moment-format closedAt 'MM/DD/YYYY hh:mm:ss'}}
{{/if}}
```

Secondly, I apologize for this, but the task was incorrectly written. We don't want a formatted timestamp, we want a "friendly" timestamp, which we get ,using `moment-from` instead of `moment-format`, so what you actually want here is 

```Handlebars
{{#if isSelf}}
  You closed this {{moment-from-now closedAt interval=60000}}
{{else}}
  {{author.username}} closed this {{moment-from-now closedAt interval=60000}}
{{/if}}
```

Similar to how the `conversation-part-comment` does it with its own timestamp.
 
This will cause the text to say something like

```
"You closed this a few seconds ago"
"Joe closed this a minute ago"
"James closed this a couple of hours ago"
"etc."
```